front/src/components/socketComment/topicInfo.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component, useCallback, useEffect, useState } from "react";
+import NavBar from "../navBar/NavBar";
+import { useLocation } from "react-router-dom";
+import Socket from "./socket";
+import { getCookiesName, getJwt } from "../helpers/userService";
+import MessageList from "./showMessage";
+import Form from "../commentsForm/form";
+import { message } from "antd";
+import TopicItem from "../topicItemComponent/topicItem";
+
+export default function TopicInfo() {
+  const ws = new WebSocket("ws://localhost:8081");
```

Comment:
move localhost to .env 
and call it like 'backendApiUri' 
and use here and everywhere when you sending requests to backend